demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -677,3 +698,157 @@ def _create_failed_packs_report(lint_status: dict, path: str):
         if path and failed_ut:
             file_path = Path(path) / "failed_lint_report.txt"
             file_path.write_text('\n'.join(failed_ut))
+
+    def create_json_output(self):
+        if not self.json_file_path:
+            return
+
+        if os.path.exists(self.json_file_path):
+            json_contents = get_json(self.json_file_path)
+
+        else:
+            json_contents = {}
+
+        for check in self.linters_error_list:
+            if check.get('linter') == 'flake8':
```

Comment:
im a bit concerned that if we add another linter in the future, then it will be missed and wont be added here
perhaps add a comment where ever it makes sense to and refer here?